PelicanPlatform/pelican
# Set correct log level for Prometheus logger

Fixes #290 

## Testing Instructions

Run your `director` then `registry` in the same container and in the terminal for `registry` you should be able to see a warning and an error log message:

```console
WARNING[2023-11-08T19:49:54Z] A lockfile from a previous execution already existed. It was replaced  component=tsdb file=/var/lib/pelican/monitoring/data/lock
INFO[2023-11-08T19:49:54Z] Stopping scrape discovery manager...          component=prometheus
INFO[2023-11-08T19:49:54Z] Stopping scraper config periodic reload...    component=prometheus
INFO[2023-11-08T19:49:54Z] Stopping scrape manager...                    component=prometheus
INFO[2023-11-08T19:49:54Z] Scrape discovery manager stopped              component=prometheus
INFO[2023-11-08T19:49:54Z] Stopping scraper config periodic reload...    component=prometheus
INFO[2023-11-08T19:49:54Z] Scrape manager stopped                        component=prometheus
ERROR[2023-11-08T19:49:54Z] opening storage failed: lock DB directory: resource temporarily unavailable  component=prometheus
```

This means that the error level from Prometheus components is propagated correctly to the log message.
